docs/source/run/parameters.rst

```diff
@@ -34,6 +34,8 @@ For a list of supported functions see the
 `AMReX documentation <https://amrex-codes.github.io/amrex/docs_html/Basics.html#parser>`__.
 Sometimes it is necessary to use double-quotes around expressions, especially when providing them
 as command line parameters. Multi-line expressions are allowed if surrounded by double-quotes.
+For input parameters of type `string` curly braces ``{}`` are needed to access ``my_constants``
+or to evaluate expressions with the parser, similar to Python f-strings.
```

Comment:
Could you elaborate just a bit here, so users could benefit from it without getting to Python f-string documentation or looking at the PR description? I believe something like this would cover the main feature. You might also add a sentence "This is useful for ..." if you have a good use case to highlight.
```suggestion
or to evaluate expressions with the parser, similar to Python f-strings. In particular, expressions inside ``{…}`` are evaluated using the parser if it is not found in my_constants (if it is found in my_constants, the variable name is pasted instead).
```